tests/components/AppliedFilters.test.tsx

```diff
@@ -1,5 +1,5 @@
-import { render, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
+import { render, screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
```

Comment:
I think we can just use the default export of this library instead of the named export here and in other test files